internal/provider/validators_test.go

```diff
@@ -267,6 +267,18 @@ func Test_validateConsulClusterTier(t *testing.T) {
 		input    string
 		expected diag.Diagnostics
 	}{
+		"development": {
+			input:    "development",
+			expected: nil,
+		},
```

Comment:
looks like the `valid tier lowercase` can be removed with this test case doing the same thing.

Reply:
I like having the additional test case. I know it is duplicated, but makes sure all tiers are covered and the different ways to spell them.